worlds/ffmq/Client.py

```diff
@@ -67,11 +78,11 @@ async def validate_rom(self, ctx):
     async def game_watcher(self, ctx):
         from SNIClient import snes_buffered_write, snes_flush_writes, snes_read
 
-        check_1 = await snes_read(ctx, 0xF53749, 1)
+        check_1 = await snes_read(ctx, 0xF53749, 6)
         received = await snes_read(ctx, RECEIVED_DATA[0], RECEIVED_DATA[1])
         data = await snes_read(ctx, READ_DATA_START, READ_DATA_END - READ_DATA_START)
-        check_2 = await snes_read(ctx, 0xF53749, 1)
-        if check_1 != b'\x01' or check_2 != b'\x01':
+        check_2 = await snes_read(ctx, 0xF53749, 6)
+        if validate_read_state(check_1, check_2) == False:
```

Comment:
This is just stylistic, but `== False` seems a little weird?
```suggestion
        if not validate_read_state(check_1, check_2):
```

Reply:
i was wondering why it wouldn't let me use `!validate_read_state(check_1, check_2)`